### PR TITLE
Add Symfony 4.0

### PR DESCRIPTION
**Pull request by rpkamp**

### CODE REVIEW

File: symfony-4.0/composer.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+    "type": "project",
+    "license": "proprietary",
```
Comment (lyrixx) [inline symfony-4.0/composer.json]: mit ?

Comment (rpkamp) [reply symfony-4.0/composer.json]: You're right. `propietary` is the default used by `composer create-project`, but indeed this is MIT, as Symfony uses MIT. I've amended my commit.

File: symfony-4.0/.gitignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (Simperfit) [inline symfony-4.0/.gitignore]: do you need this blank line ?

Comment (rpkamp) [reply symfony-4.0/.gitignore]: This is just the default `.gitignore` that is generated when creating a minimal symfony installation.

I've amended my commit though and removed the blank line.

**Comment by rpkamp** [comment]
Seems nothing is happening here even though this PR has been approved.

@kenjis @lyrixx @javiereguiluz Anything I need to do for this branch to be merged?

**Comment by javiereguiluz** [comment]
@rpkamp I'd say the work is finished ... but @kenjis needs to find the time to review this and, if he considers appropriate, merge it. Please, let's not put more pressure on open source project maintainers 🙏. They do the best they can! Cheers.

**Comment by rpkamp** [comment]
Ok cool, just checking you weren't waiting for anything I'd missed, basically completely stalling this PR :+1:

**Comment by kenjis** [comment]
Sorry, I have no time to review it for a while. 
Because I have no PHP 7 benchmarking environment now, and I am very busy.

But it seems the code is okay and 2 people approved already, so I merge this PR.
Thank you for PR. Thank you who reviewed this.

**Pull request merged**